datadog_checks_base/datadog_checks/base/data/agent_requirements.in

```diff
@@ -24,7 +24,8 @@ foundationdb==6.3.18; python_version > "3.0"
 futures==3.3.0; python_version < "3.0"
 gearman==2.0.2; sys_platform != "win32" and python_version < "3.0"
 immutables==0.15; python_version > "3.0"
-in-toto==1.0.1
+in-toto==1.0.1; python_version < "3.0"
+in-toto==1.1.1; python_version > "3.0"
```

Comment:
Should this be >= instead of >?

Reply:
No, 3.8 > 3.0

Reply:
OK, but what if you are at 3.0?

Reply:
We do this for all other [requirements](https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/data/agent_requirements.in), as python 3.0 is not used. It wouldn't hurt to add the `=`, though, if you think it's worth it

Reply:
I won't insist on it, since it's best to be consistent one way or another. Thanks!